.github/workflows/ci-california-api.yml

```diff
@@ -1,4 +1,4 @@
-name: CI - FastAPI Template
+name: CI - California API Run Checks
 
 on:
   workflow_call:
```

Comment:
Did we mean for this to be `workflow_dispatch`? Or are we calling it from another workflow?